package.json

```diff
@@ -71,18 +72,23 @@
     "defined": "1.0.0",
     "express": "4.14.0",
     "history": "2.1.2",
+    "inuitcss": "^6.0.0-beta.3",
     "isomorphic-fetch": "2.2.1",
     "lodash": "4.15.0",
+    "moment": "2.15.1",
```

Comment:
Vi har brukt date-fns for datohåndtering hittil. Ville foretrekke å bruke det. Se: https://github.com/NDLANO/ndla-frontend/blob/master/src/util/formatDate.js